zeebe/engine/src/main/java/io/camunda/zeebe/engine/state/mutable/MutableUsageMetricState.java

```diff
@@ -19,5 +20,5 @@ public interface MutableUsageMetricState extends UsageMetricState {
 
   void resetActiveBucket(final long fromTime);
 
-  void updateActiveBucketTime(long resetTime);
+  PersistedUsageMetrics getOrCreateActiveBucket();
```

Comment:
💭 does this have to be a public interface methods? It might just be used within the state in the various `record*` methods 🤔